src/scss/Triggers.scss

```diff
@@ -31,3 +31,9 @@ limitations under the License.
     margin-top: $spacing-03
   }
 }
+
+.triggers--detail-block {
+  margin-bottom: $layout-01;
+  padding: $layout-01;
+  background-color: #ffffff;
```

Comment:
is there a suitable Carbon token we could use here? https://www.carbondesignsystem.com/guidelines/color/usage/
maybe `$ui-02` container background?